packages/opentelemetry/src/spanExporter.ts

```diff
@@ -133,7 +133,9 @@ export class SentrySpanExporter {
     const remainingOpenSpanCount = finishedSpans.length - sentSpanCount;
 
     DEBUG_BUILD &&
-      logger.log(`SpanExporter exported ${sentSpanCount} spans, ${remainingOpenSpanCount} unsent spans remaining`);
+      logger.log(
+        `SpanExporter exported ${sentSpanCount} spans, ${remainingOpenSpanCount} spans are waiting for their root span to finish`,
```

Comment:
hmm, what about:

```suggestion
        `SpanExporter exported ${sentSpanCount} spans, ${remainingOpenSpanCount} spans are waiting for their parent spans to finish`,
``` 

or `ancestor spans` which is _really_ correct but a bit of a weird term 😅 

because this is actually correct? E.g. if a parent somewhere is missing, this will forever remain pending like this (until it is cleaned up) even if the root was actually already sent 🤔